### PR TITLE
Fix: unskip authorizer tests

### PR DESCRIPTION
**Pull request by YarikRevich**
Associated PRs:
- https://github.com/0chain/actions/pull/184
- https://github.com/0chain/0helm/pull/311
- https://github.com/0chain/actions/pull/218

### CODE REVIEW

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Please fix TestBridgeBurn/Get_WZCN_burn_ticket,_should_work, happy to merge after it is passing

**Comment by YarikRevich** [comment]
This PR is blocked by the lack of graph-node deployment for system tests

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
           sudo apt-get -y install build-essential nghttp2 libnghttp2-dev libssl-dev
       - uses: actions/setup-go@v3
         with:
-          go-version: '1.20'
+          go-version: '1.18'
```
Comment (Kishan-Dhakan) [inline .github/workflows/ci.yml:63]: Please revert, lint issues are fixed.

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Review by stewartie4 — COMMENTED** [review]
@YarikRevich what is the status of this PR?

**Comment by Kishan-Dhakan** [comment]
@YarikRevich it's https://github.com/0chain/system_test/issues/717 this one, are you working on it?

**Comment by YarikRevich** [comment]
@Kishan-Dhakan Yes, I'm working on IPFS deployment issue

File: tests/cli_tests/zwalletcli_zcnbridge_burn_test.go [hunk 2]
```diff
@@ -19,21 +19,18 @@ import (
 
 func TestBridgeBurn(testSetup *testing.T) {
 	t := test.NewSystemTest(testSetup)
-	t.Skip("skip till authorizers are re-enabled")
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zwalletcli_zcnbridge_burn_test.go:22]: Please skip all the other zcnbirdge tests as well

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Review by Kishan-Dhakan — APPROVED** [review]
LGTM, happy to merge pending passing tests

**Comment by Kishan-Dhakan** [comment]
Passing run: https://github.com/0chain/system_test/actions/runs/5754104281/job/15599526710

image

**Pull request merged**